include/deal.II/fe/fe_values.h

```diff
@@ -2633,7 +2633,7 @@ class FEValuesBase :
    * keep around between subsequent calls to reinit() in order to compute the
    * cell similarity.
    */
-  boost::signals2::connection tria_listener;
+  std::vector<boost::signals2::connection> tria_listener;
```

Comment:
I will now use one `tria_listener_refinement` and one `tria_listener_mesh_transform` in the code I will soon commit (I'm just running through the test suite to make sure that everything works now).